### PR TITLE
[WIP] Update VT incentive relationships

### PR DESCRIPTION
**Pull request by matelau**
This nested prerequisite was giving me issues in incentive admin when trying to seed the data with the existing data. I wanted to check if the relationship as defined is equivalent to the pre-existing nested relationship.

### CODE REVIEW

File: data/VT/incentive_relationships.json [hunk 1]
```diff
@@ -14,10 +14,9 @@
     "VT-86": "VT-84",
     "VT-89": "VT-87",
     "VT-93": "VT-91",
+    "VT-94": {"anyOf": ["VT-75", "VT-77", "VT-79", "VT-81"]},
     "VT-96": {
-      "allOf": [
-        "VT-94",
-        { "anyOf": ["VT-75", "VT-77", "VT-79", "VT-81"] }]
+      "allOf": ["VT-94"]
```
Comment (matelau) [inline data/VT/incentive_relationships.json:19]: Another alternative here could be something like, which would also work downstream in incentive admin... 
```json
    "VT-96": {
      "allOf": ["VT-94"],
      "anyOf": ["VT-75", "VT-77", "VT-79", "VT-81"]
    }
```

Comment (matelau) [reply data/VT/incentive_relationships.json:19]: But, if we go this route we would also need to update the schema to allow for more than oneOf the expected definitions. 
image

Comment (matelau) [reply data/VT/incentive_relationships.json:19]: Thinking about this more and I don't think either solution works. I'm going to consider if I can create a recursive solution in Incentive Admin to follow the nesting. Open to ideas though. 
image

**Comment by oyamauchi** [comment]
So VT-96 is all cleaned up now, but the schema simplification may still be relevant? Or maybe not, since ideally we won't be bringing in significant amounts of new incentive data before the admin tool is online?

**Comment by matelau** [comment]
> So VT-96 is all cleaned up now, but the schema simplification may still be relevant? Or maybe not, since ideally we won't be bringing in significant amounts of new incentive data before the admin tool is online?

I hope not 🤞🏼! I'll close this PR and we can re-visit if necessary.

**Pull request closed without merging**